### ISSUE TITLE
can't access "worksheet: evaluate worksheet"

### ISSUE THREAD

**Issue by cmpscabral** [jcartledge/sublime-worksheet#21]
Hi,
I just installed this using package manager, but using ctrl+shift+P and searching for the "woksheet: evaluate worksheet" command, I can't find it.

I tried restarting sublime but the error remains.

I'm on ubuntu 11.10 (oneiric) 64bit.

Thanks

**Comment by j-hernandez** [comment]
Confirmed on Ubuntu 12.04. Relevant log entries from ST2 console: 

```
Reloading plugin /home/james/.config/sublime-text-2/Packages/Worksheet/__init__.py
Reloading plugin /home/james/.config/sublime-text-2/Packages/Worksheet/worksheet.py
Traceback (most recent call last):
  File "./sublime_plugin.py", line 62, in reload_plugin
  File "./worksheet.py", line 3, in <module>
    import repl
  File "./repl/__init__.py", line 1, in <module>
    from repl import get_repl
  File "./repl/repl.py", line 1, in <module>
    import pexpect
  File "./repl/pexpect.py", line 85, in <module>
    support it. Pexpect is intended for UNIX-like operating systems.""")
ImportError: No module named resource

A critical module was not found. Probably this operating system does not
support it. Pexpect is intended for UNIX-like operating systems.
```

**Comment by cmpscabral** [comment]
yep, forgot to check the console :) I get the exact same error

**Comment by toxinu** [comment]
Same for me.

**Comment by inactivist** [comment]
Same here.  Ubuntu 12.04 64 bit.  Not sure if this matters, but I launched Sublime Text 2 from inside an active Virtualenv created with the  `--no-site-packages` option.

Edit: Nope, starting Sublime Text 2 while no virtualenv active doesn't fix the problem.

**Comment by inactivist** [comment]
Derp... I forgot, ST2 has its own private Python interpreter rolled in.

``` python
>>> import sys
>>> sys.version
'2.6.6 (r266:84292, Jun 16 2011, 22:27:44) \n[GCC 4.4.5]'
```

**Comment by cronco** [comment]
Yup, happens to me too, Ubuntu 13.04.

**Comment by jcartledge** [comment]
Thanks all - I'll try to check this out today.

**Comment by jcartledge** [comment]
Sorry Linux people, my bad. For some reason the resource module (a pexpect dependency) isn't packaged in ST2. Luckily it's only used for one call which can be replaced by shelling out to ulimit. Fix is in 0.7.3 which should be in Package Control soon...

**Comment by j-hernandez** [comment]
Much appreciated, looking forward to 0.7.3!
